### DISCUSSION TITLE
Lesson 2: Conflict, Political Science & Information Flows - Reference Video Missing.

### DISCUSSION THREAD

**Discussion by LEONOB2014** [OpenMined/courses#92] [Feedback] [answered]
Near the end of the video, Andrew referenced a segment video discussing the conflict between India and Pakistan, but such a segment is missed. I guess is related to a glitch in editing.

**Comment by iamtrask** [comment]
I'm on it - should be able to have the fix up in a couple hours. Sorry it's not there - it's an absolutely incredible story.

**Reply by mcleonard** [reply]
Tracking this here: https://github.com/OpenMined/courses/issues/98

Can you close the issue when it's fixed? Thanks 😄

**Comment by iamtrask** [comment, marked as answer]
Fixed

**Reply by LEONOB2014** [reply]
Thank you so much.
